src/apb/engine.py

```diff
@@ -452,6 +452,8 @@ def relist_service_broker(kwargs):
                        base64.b64encode("{0}:{1}".format(kwargs['basic_auth_username'],
                                                          kwargs['basic_auth_password']))
                        }
+        elif kwargs['auth_token'] is not None:
```

Comment:
I notice we have duplicate code here. Thoughts on breaking out this logic to be something like `cluster_request` and then in functions like `relist_service_broker` and `broker_request` we can call this function to instantiate the headers.

Just a nit :)

Reply:
Agreed there is duplication that's worth dealing with. But I barely know python syntax as well I don't know the `apb` codebase so refactoring anything would require me to put put quiet some time on it and I don't have it atm :confused:

Reply:
I think when we rewrite the apb tooling we can incorporate this. I'm fine with it as is